test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -691,7 +691,7 @@ func BenchmarkPerfScheduling(b *testing.B) {
 					for feature, flag := range tc.FeatureGates {
 						defer featuregatetesting.SetFeatureGateDuringTest(b, utilfeature.DefaultFeatureGate, feature, flag)()
 					}
-					results := runWorkload(ctx, b, tc, w)
+					results := runWorkload(ctx, b, tc, w, false)
```

Comment:
> Each benchmark test case runs with a fresh etcd instance. Therefore it is not
necessary to delete objects after a run.

yes, it's a fresh etcd instance, but I think it's always a good habit to do the cleanup after the test, this is also true for us to delete the object before removing the namespace entirely, it's not necessary but it's not something we should get rid of.

> A future unit test might reuse etcd, therefore cleanup is optional.

I don't think a future ut should reuse the leftover etcd instance.

Reply:
Each perf test starts with a clean slate (new etcd, apiserver, scheduler). Therefore cleaning up *after* the tests is unnecessary and just slows down running them. Starting etcd and apiserver again then takes time, but that's justified because of the "clean slate".

With integration tests it's the opposite: the cost of starting etcd and apiserver gets amortized over multiple tests, but then each test has to clean up properly. That's okay because the workloads selected for integration testing tend to create less objects.

Reply:
Ack, thanks for the explanation!